packages/earljs/src/Expectation.ts

```diff
@@ -81,6 +82,38 @@ export class Expectation<T> {
     }
   }
 
+  /**
```

Comment:
inline documentation is great but we need to these to API reference in docs directory too.